backend/src/main/kotlin/app/ehrenamtskarte/backend/common/utils/Sentry.kt

```diff
@@ -0,0 +1,17 @@
+package app.ehrenamtskarte.backend.common.utils
+
+import io.sentry.Sentry
+
+fun initializeSentry() {
+    Sentry.init { options ->
+        options.dsn = "https://c0c177b8fa35e0fd90d788b0d02bcb2c@sentry.tuerantuer.org//7"
+        // https://docs.sentry.io/concepts/key-terms/tracing/
+        options.tracesSampleRate = 0.05
+    }
+}
+
+fun reportErrorToSentry(e: Exception) {
+    if (Sentry.isEnabled()) {
+        Sentry.captureException(e)
+    }
+}
```

Comment:
🔧  This function is unnecessary, we can just use `Sentry.captureException(e)`

Reply:
@andrew8er but is this function checking if `Sentry.isEnabled()` ? Couldn't find anything in the docs about that